### ISSUE TITLE
iOS: using offlineAnalysis on a file in the app's bundle.

### ISSUE THREAD

**Issue by Sjakelien** [superpoweredSDK/Low-Latency-Android-Audio-iOS-Audio-Engine#138]
Hi, 
I have modified the OfflineProcessingExample to process a file that resides in my app's bundle.
My viewDidLoad now looks as follows:

```
- (void)viewDidLoad {
    [super viewDidLoad];
   NSURL *myURL = [[NSBundle mainBundle]
                    URLForResource: @"test" withExtension:@"wav"];
    
    [self offlineAnalyze:myURL];
}
```

However, when running the app, both in the Simulator as well as on a device, I get the following error:
"open error: Can't open this file"
The path  of the NSURL ("file:///Users/me/Library/Developer/CoreSimulator/Devices/5AC2740B-8B51-4955-B85F-939AF76B9C76/data/Containers/Bundle/Application/58773BBB-AEB4-49B0-B371-F1FDF7C8EE55/SuperpoweredOfflineProcessingExample.app/test.wav") is valid, and as far as I understand, a Wave file is a valid file type as well.

What am I doing wrong?

**Comment by superpoweredSDK** [comment]
Perhaps that wave file has a strange encoding. Try with PCM and 16-bit samples first.

**Comment by Sjakelien** [comment]
I have opened an .mp3 in Audacity and exported it as "WAV (Microsoft) signed 16 bit PCM".  I also tried the .mp3, as well as .aiff. Same result.
Deliberately changing the name of the file in the code results in "open error: Path is null" so I assume that the path is OK.

**Comment by Sjakelien** [comment]
But basically, you acknowledge that my code is OK?

**Comment by superpoweredSDK** [comment]
Yes, it looks okay. I wonder what happens if you try to copy that file to a temporary space, and try from there?

**Comment by Sjakelien** [comment]
Sorry for my late reply. I did what you suggested, but the results are the same: "open error: Can't open this file". To be honest, I would have been surprised, if it would have worked.
Since my code is just a very simple adjustment to your project, I would encourage you to make these changes, and see if it works for you.

What I did:
-From the Finder, I dragged an audio file to my project, with "copy items if needed" checked. Also, the "Add to targets" checkbox is checked.
-I verified newly added file's target membership
-Made the above changes to ViewDidLoad, outcommenting the existing lines of that method.
